frontend/src/pages/SearchResultsPage.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Container, Typography } from '@material-ui/core';
+import React, { ReactElement } from 'react';
+
+// import NotFoundIcon from '../assets/not-found.svg';
+// import styles from './NotFoundPage.module.css';
+
+const SearchResultsPage = (): ReactElement => {
+  return (
+    <Container maxWidth="md">
+      <Box pb={3} textAlign="center">
+        {/* <img src={NotFoundIcon} alt="CU Apts Logo" className={styles.NotFoundImage} /> */}
+        <Typography variant="h5">EMPTY NOW!</Typography>
```

Comment:
We will want to be able to access the query (what the user typed into the search bar) so that we can fetch the results for that query from the backend later. 

We can use `useSearchParams` from `react-router-dom` to access the query. See [this react documentation](https://reactrouter.com/docs/en/v6/api#usesearchparams) for how to use it.

Reply:
For now, we can make sure we are getting the correct query by displaying the result on the page
```
<Typography variant="h5">`EMPTY NOW! showing results for {getParams.get('q')}`</Typography>
```